mov_cli/subtitles.py

```diff
@@ -0,0 +1,44 @@
+import requests
+
+
+class Subtitles:
+    """
+    pull subtitles from api.opensubtitles.com and return
+    a link to the .srt file which can be passed into vlc or mpv
+    """
+
+    def __init__(self, key, language="en"):
```

Comment:
Can `key` be optional? E.g. be `None` if the user provides no API key.

Reply:
not really, the new opensubtitles api forces keys.. no anonymous downloads! Still, we could create an account for an organization but it still has limits.

Reply:
also I was thinking the key would be inputted into the config file, not sure how to do that

Reply:
> not really, the new opensubtitles api forces keys.. no anonymous downloads! Still, we could create an account for an organization but it still has limits.

ah okay

Reply:
> also I was thinking the key would be inputted into the config file, not sure how to do that

I can do that, we just need to add it as a field in the config class and then pass the config class as a parameter to the subtitles class.